WolvenKit.App/ViewModels/Documents/RedDocumentViewModel.cs

```diff
@@ -229,14 +234,19 @@ protected void AddTabForRedType(RedBaseClass cls)
 
         private void PopulateItems()
         {
-            TabItemViewModels.Add(new RDTDataViewModel(Cr2wFile.RootChunk, this));
+            var root = new RDTDataViewModel(Cr2wFile.RootChunk, this);
+            root.FilePath = "(root)";
+            TabItemViewModels.Add(root);
             AddTabForRedType(Cr2wFile.RootChunk);
 
             foreach (var file in Cr2wFile.EmbeddedFiles)
             {
                 if (file.Content != null)
                 {
-                    TabItemViewModels.Add(new RDTDataViewModel(file.Content, this));
+                    var vm = new RDTDataViewModel(file.Content, this);
+                    vm.FilePath = file.FileName;
```

Comment:
I think you can do something like
```suggestion
                    var vm = new RDTDataViewModel(file.Content, this)
                    {
                                        FilePath = file.FileName,
                                        IsEmbeddedFile = true
                    };
```